hw/ip/hmac/data/hmac.prj.hjson

```diff
@@ -7,6 +7,7 @@
     version:            "0.5",
     life_stage:         "L1",
```

Comment:
Do you want to fully sign this off? If so, this should move to L2 for Sign Off

Reply:
Yes I would like to fully sign this off, will update the change, thanks for the reminder